lxd/lxd.spec

```diff
@@ -89,6 +90,12 @@ Suggests: edk2-ovmf
 Suggests: genisoimage
 Suggests: qemu-img
 Suggests: qemu-system-x86-core
+Suggests: qemu-ui-spice-core
+Suggests: qemu-audio-spice
+Suggests: qemu-char-spice
+Suggests: qemu-device-usb-redirect
+Suggests: qemu-device-display-virtio-vga
+Suggests: qemu-device-display-virtio-gpu
```

Comment:
Seems that these packages are new in Fedora 34. What are they used for? Are they required to run LXD virtual machine instances? I cannot find any references of audio devices being created by LXD. I'd like to only list packages that are functionally required for virtual machine support here.

Reply:
Yes, they were required for me to start a VM.